app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -62,7 +62,7 @@ def new
         collection_type_id = params[:collection_type_id]
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.dashboard.collections.new.header'), request.path
+        add_breadcrumb t(:'hyrax.dashboard.collections.new.header') + " " + @collection.collection_type.title, request.path
```

Comment:
PR #2071 is fixing the breadcrumbs.  I'd prefer not to mix these two PRs.  Can you remove this change from here?

Reply:
Okay, reverted.